src/M6Web/Component/RedisMock/RedisMock.php

```diff
@@ -1211,6 +1211,48 @@ public function zunionstore($destination, array $keys, array $options = array())
         return $this->zcount($destination, '-inf', '+inf');
     }
 
+    /**
+     * Mock the `zscan` command
+     * @see https://redis.io/commands/zscan
+     * @param string $key
+     * @param int $cursor
+     * @param array $options contain options of the command, with values (ex ['MATCH' => 'st*', 'COUNT' => 42] )
+     * @return $this|array|mixed
+     */
+    public function zscan($key, $cursor, $options = [])
+    {
+        $options = array_change_key_case($options, CASE_UPPER); // normalize to match Laravel/Symfony
+        $count   = isset($options[ 'COUNT' ]) ? (int)$options[ 'COUNT' ] : 10;
+        $match   = isset($options[ 'MATCH' ]) ? $options[ 'MATCH' ] : '*';
+        $pattern = sprintf('/^%s$/', str_replace(['*', '/'], ['.*', '\/'], $match));
+
+        $iterator = $cursor;
+
+        if (!isset(self::$dataValues[$this->storage][$key]) || $this->deleteOnTtlExpired($key)) {
+            return $this->returnPipedInfo([0, []]);
+        }
+
+        $set = self::$dataValues[ $this->storage ][ $key ];
```

Comment:
Just a small fix for the code style.

```suggestion
        $set = self::$dataValues[$this->storage][$key];
```